diff --git a/cpp/jni/javet_v8_runtime.cpp b/cpp/jni/javet_v8_runtime.cpp
index a4e6bfb5..f173be97 100644
--- a/cpp/jni/javet_v8_runtime.cpp
+++ b/cpp/jni/javet_v8_runtime.cpp
@@ -370,7 +370,7 @@ namespace Javet {
             // nodeIsolateData->set_is_building_snapshot(createSnapshotEnabled);
             node::crypto::InitCryptoOnce(v8Isolate);
         }
-        v8Isolate->SetModifyCodeGenerationFromStringsCallback(nullptr);
+        //v8Isolate->SetModifyCodeGenerationFromStringsCallback(nullptr);
 #else
         if (createSnapshotEnabled) {
             v8Isolate = v8::Isolate::Allocate();
-- 
2.20.1

